### PR TITLE
DEVSU-2193 Print Report Styling

### PR DESCRIPTION
**Pull request by bnguyen-bcgsc**
- Reformat key alteration categories in print table
- Change 'captiv 8' spelling to 'CAPTIV-8'
- Realign logo with other components in beta printview
- Increase report title font size in beta printview
- Add separating commas between key alterations variants

### CODE REVIEW

**Review by elewis2 — CHANGES_REQUESTED** [review]
remove final comments in variant lists in key alterations table

File: app/components/SummaryPrintTable/index.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-nested-ternary */
```
Comment (kttkjl) [inline app/components/SummaryPrintTable/index.tsx:1]: you know the rules and so do i

**Review by elewis2 — APPROVED** [review]

File: app/components/SummaryPrintTable/index.tsx [hunk 2]
```diff
@@ -12,6 +12,15 @@ type SummaryPrintTableProps = {
   renderValue?: (value: any) => unknown;
 };
 
+const variantTypes = ['cnv', 'smallMutation', 'structuralVariant', 'expression'];
+
+const variantToStrings: Dictionary<string> = {
```
Comment (kttkjl) [inline app/components/SummaryPrintTable/index.tsx:17]: weird, was the import from lodash needed? I thought the type inferrence by typescript was enough

**Review by kttkjl — APPROVED** [review]
Code looks good, but not sure about the import type from lodash

File: app/components/SummaryPrintTable/index.tsx (hunk 2 above)
Comment (bnguyen-bcgsc) [reply app/components/SummaryPrintTable/index.tsx:17]: oh yea I tried the native typescript dict but it did not let me index with strings so I had to use the lodash dict

**Pull request merged**